### ISSUE TITLE
Should be an "or" condition instead of "and"

### ISSUE THREAD

**Issue by sradco** [linux-system-roles/logging#6]
https://github.com/linux-system-roles/logging/blob/a72b50f6f686fb071d84d62e78bd6a6df456f79a/roles/rsyslog/tasks/main.yaml#L134

@nhosoi Hi, Should the condition here be with an "or" instead of "and"?

**Comment by sradco** [comment]
Doesn't  item.state|d() mean default is empty? I think for is defined it should be item.state is defined.

**Comment by nhosoi** [comment]
Thank you, @sradco !  Definitely, there was a bug...
Could your review #7 including this commit?
https://github.com/linux-system-roles/logging/pull/7/commits/cd28ff8473fbf8e073adce00093a78f35a2152c2
I tested starting with:
```
rsyslog__example: true
rsyslog__capabilities: [ 'network', 'remote-files', 'tls' ]
```
to
```
rsyslog__viaq: true
rsyslog__capabilities: [ 'viaq', 'viaq-k8s' ]
```
to
```
rsyslog__viaq: true
rsyslog__capabilities: [ 'viaq' ]
```
then none.  I verified that by running the ansible command, necessary files are created and unnecessary files are removed based on the combination of rsyslog config type and capabilities.
